tensorflow/compiler/tests/image_ops_jit_compile_test.py

```diff
@@ -50,9 +55,9 @@ def train(x, y):
         return grads
 
       compiled_train = def_function.function(train, jit_compile=True)
-      x = array_ops.zeros((1, img_width // 2, img_width // 2, 1),
+      x = random_ops.random_uniform((1, img_width // 2, img_width // 2, 1),
                           dtype=dtypes.float32)
-      y = array_ops.zeros((1, img_width, img_width, 1), dtype=dtypes.float32)
+      y = random_ops.random_uniform((1, img_width, img_width, 1), dtype=dtypes.float32)
```

Comment:
Using equal input and labels will always result in zero gradients so this test would pass even if the gradient computation is wrong.

Reply:
Please fix this then?

Reply:
> Please fix this then?

Sorry, I think I wasn't clear in the description. Changing from `tf.zeros` to `tf.random.random_uniform` fixes this issue.